monitors.yml

```diff
@@ -1,24 +1,142 @@
 defaults:
   monitors:
-    multi: true
     options:
       locked: true
+      new_host_delay: 300
+      no_data_timeframe: 2
+      notify_audit: false
+      renotify_interval: 0
+      silenced: {}
+      timeout_h: 0
```

Comment:
These were all the ones that were common in the generated output. These all exist from 'Datadog-builder init'